document/core/valid/instructions.rst

```diff
@@ -566,7 +566,7 @@ Aggregate Reference Instructions
 
 .. _valid-array.set:
 
-:math:`\ARRAYSET~x~i`
+:math:`\ARRAYSET~x`
 .....................
```

Comment:
Can you also adjust the underline? Otherwise Sphinx will complain.

Reply:
Oops, forgot it. Thanks.